src/i18n/nl/index.ts

```diff
@@ -185,10 +185,10 @@ const nl = {
       BADGES: 'Badges',
     },
     EMPTY_CREDENTIALS: {
-      TITLE: '',
-      SUBTITLE: '',
+      TITLE: 'Het is hier een beetje stil',
```

Comment:
Ik zou dit iets formeler weergeven: Geen credentials gevonden. Ik denk dat wat beter is voor onze doelgroep

Reply:
Goede suggestie! De engelse versie is misschien wat informeler:
 TITLE: "It's a bit quiet in here",
      SUBTITLE: 'Why not add some credentials to start your new digital me?',
      
Weet niet of we dat ook in het Nederlands willen of wat formeler willen weergeven?

Reply:
Hmm ja ik vind het in het engels sowieso wat minder irritant overkomen, komt misschien qua taal beter bij past ofzo ;). @JelleMillenaar wat vind jij?

Reply:
Personally to me `'Het is hier een beetje stil'` is a great translation here from the original English version that came from the UX designs.

In general if the original English version is informal, imo all the translations should reflect the informalness as much as possible.